spec/swagger_helper.rb

```diff
@@ -18,6 +18,44 @@
       info: {
         title: "Teaching Vacancies ATS API",
         version: "v1",
+        description: <<~DESCRIPTION,
+          This document outlines the API’s key features, the data required for each listing,
+          and the steps needed to set up an integration.
+
+          This documentation describes each endpoint’s request parameters, response formats, and possible errors,
+          so you can seamlessly integrate Teaching Vacancies into your ATS workflow.
+
+          ## Introduction
+
+          The **Teaching Vacancies ATS API** enables you to manage job listings on behalf of schools or trusts
+          through your own Applicant Tracking System (ATS) or HR software.
+
+          By calling this API, you can:
+
+          - **List** all your active vacancies (with pagination)
+          - **Retrieve** details for a single vacancy
+          - **Create** new vacancies
+          - **Update** existing vacancies
+          - **Delete** vacancies when they’re no longer needed
+
+          ## Authentication
+
+          Each request requires a valid API key in the `X-Api-Key` header to ensure only authorised
+          clients can manage vacancies.
+
+          Include this key in the X-Api-Key header of each request.
+          If the key is missing or invalid, the API will respond with an HTTP 401 (Unauthorized) status.
+
+          This ensures that only approved clients can create, update, or remove job listings.
+          If you ever need a new or replacement key, let us know, and we’ll assist you with the process.
```

Comment:
Should we add an email address linking to `let us know` ? There was a shared inbox email (that I don't have access to ATM) that we could use here. Will check tomorrow with the team.